src/addressResolvers/index.ts

```diff
@@ -12,7 +13,13 @@ import {
 import { Address, Handle } from '../utils';
 import { timeAddressResolverResponse as timeResponse } from '../helpers/metrics';
 
-const RESOLVERS = [ensResolver, unstoppableDomainResolver, lensResolver, starknetResolver];
+const RESOLVERS = [
+  snapshotResolver,
```

Comment:
What happens when the user edits his profile? 🤔 we will still have it in our cache right

Reply:
Good point … what to do

Reply:
Ideally the sequencer would trigger stamp for clear cache of that specific user who updated his profile

Reply:
Cache clearing for `address` added. will make a PR on sequencer side to call it on user profile update